### PR TITLE
Proposal to rework the introduction

### PR DESCRIPTION
**Pull request by boucadair**
simplify, avoid redundant statements, better readability flow, soften actn context, etc.

### CODE REVIEW

File: draft-ietf-ivy-network-inventory-yang.md [hunk 1]
```diff
@@ -111,64 +111,37 @@ YANG data model that is application- and technology-agnostic.  The
...
+management which was specified many years ago. Network inventory management is a critical
+part for ensuring that the network remains healthy (e.g., auditing to identify faulty elements), well-planned (e.g., identify assets to upgrade or to decommission), and maintained appropriately to meet the performance objectives. Also, network inventory
+management allows operators to keep track of which devices are deployed in their networks, including relevant embedded software and
 hardware versions.
 
-The Network Inventory management also helps the operator to
-know when to acquire new assets and what is needed, or to
-decommission old or faulty ones, which can help to improve network
-performance and capacity planning.
+Exposing standards interfaces to retrieve and query network elements capabilities as recorded in an inventory are key enablers for many applications that consumes network inventory data. From that standpont and given the emergence of standard data models and their deployment by operators, the conventional function of inventory management is also required to be defined as a data model.
```
Comment (prmanna) [inline draft-ietf-ivy-network-inventory-yang.md:120]: spelling mistake - standpont -> standpoint

Comment (boucadair) [inline draft-ietf-ivy-network-inventory-yang.md:120]:
```suggestion
Exposing standards interfaces to retrieve and query network elements capabilities as recorded in an inventory are key enablers for many applications that consumes network inventory data. From that standpoint and given the emergence of standard data models and their deployment by operators, the conventional function of inventory management is also required to be defined as a data model.
```

Comment (boucadair) [reply draft-ietf-ivy-network-inventory-yang.md]: Thanks @prmanna. Fixed now in https://github.com/ietf-ivy-wg/network-inventory-yang/pull/53/commits/2dcb9076d22c3acbaf004684f3cc2d642b629e66

**Review by prmanna — APPROVED** [review]

**Review by YuChaode — APPROVED** [review]

**Review by italobusi — APPROVED** [review]
The text proposes some changes to text originally contributed by @jbouqui153 so I would suggest him to review these proposed changes as well before merging the PR

**Review by jbouqui153 — APPROVED** [review]

File: draft-ietf-ivy-network-inventory-yang.md [hunk 2]
```diff
@@ -111,64 +111,37 @@ YANG data model that is application- and technology-agnostic.  The
...
+management which was specified many years ago. Network inventory management is a critical
+part for ensuring that the network remains healthy (e.g., auditing to identify faulty elements), well-planned (e.g., identify assets to upgrade or to decommission), and maintained appropriately to meet the performance objectives. Also, network inventory
+management allows operators to keep track of which devices are deployed in their networks, including relevant embedded software and
 hardware versions.
 
-The Network Inventory management also helps the operator to
-know when to acquire new assets and what is needed, or to
-decommission old or faulty ones, which can help to improve network
-performance and capacity planning.
+Exposing standards interfaces to retrieve and query network elements capabilities as recorded in an inventory are key enablers for many applications that consumes network inventory data. From that standpoint and given the emergence of standard data models and their deployment by operators, the conventional function of inventory management is also required to be defined as a data model.
```
[the remark this answers is not in the document]
Comment (boucadair) [reply draft-ietf-ivy-network-inventory-yang.md]: Fixed

File: draft-ietf-ivy-network-inventory-yang.md [hunk 3]
```diff
@@ -111,64 +111,37 @@ YANG data model that is application- and technology-agnostic.  The
...
-include some technology-specific inventory details together with specific HW or SW component's attributes.
+of using vendors proprietary APIs.
 
-{{!RFC8348}} defines a YANG data model for the management of the hardware on a single server and therefore it is more applicable to the domain controller South Bound Interface (SBI) towards the network elements rather than at the domain controller's northbound. However, the YANG data model defined in {{!RFC8348}} has been used as a reference for defining the YANG network inventory data model presented in this draft.
-
-The proposed YANG data model has been analyzed at the present stage to cover the common requirements for both HW and SW use cases for Network Inventory.
+{{!RFC8348}} defines a YANG data model for the management of the hardware on a single server and therefore it is more applicable to the domain controller towards the network elements rather than at the domain controller interface to upper applications. However, the YANG data model defined in {{!RFC8348}} has been used as a reference for defining the YANG network inventory data model presented in this document.
```
[the remark this answers is not in the document]
Comment (boucadair) [reply draft-ietf-ivy-network-inventory-yang.md:130]: Geocoordinate interfaces are really vague and not helpful (the north can also be the south of another entity).

File: draft-ietf-ivy-network-inventory-yang.md [hunk 4]
```diff
@@ -111,64 +111,37 @@ YANG data model that is application- and technology-agnostic.  The
...
-definition of {{?RFC8969}},the network inventory model is a network
+definition of {{?RFC8969}}, the network inventory model is a network
 model.
 
 This document defines one YANG module "ietf-network-inventory" in {{ni-yang}}.
+This base data model is technology-agnostic (that is, valid for IP/MPLS, optical, and
+microwave networks in particular) and can be augmented to
+include required technology-specific inventory details together with specific hardware or software component's attributes.
+
+The YANG data model defined in the document is scoped to cover the common requirements for both hardware and software (but with base fucntions) use cases for Network Inventory.
```
[the remark this answers is not in the document]
Comment (boucadair) [reply draft-ietf-ivy-network-inventory-yang.md]: Fixed

File: draft-ietf-ivy-network-inventory-yang.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (italobusi) [reply draft-ietf-ivy-network-inventory-yang.md:130]: **2024-10-02 Call**

c/the domain controller interface to upper applications/the northbound interface of a network controller (e.g., toward an application or another hierarchical network controller)/

[the remark this answers is not in the document]
Comment (boucadair) [reply draft-ietf-ivy-network-inventory-yang.md]: I'm not fun of north thing, but changed the text as you suggested.

**Pull request merged**